web/App.vue

```diff
@@ -148,6 +148,10 @@ export default {
   & .v-tooltip__content {
     background: rgba(33, 33, 33, 0.9);
     font-size: 1rem;
+
```

Comment:
we don't actually need this - it was added to support the link to Vision Zero materials, but the tooltip interaction is such that clicking on links in tooltips is hard anyways, so the link was ultimately removed from the tooltip copy.